application.js

```diff
@@ -26,6 +26,7 @@ app.use(mbaasExpress.fhmiddleware());
 
 // fhlint-begin: custom-routes
 app.use('/try', require('./lib/try.js')());
+app.use('/tester', require('./lib/tester.js')());
```

Comment:
This is for purposes of dev testing. You can use tester to create any response you want.